src/flag_gems/ops/where.py

```diff
@@ -0,0 +1,16 @@
+import triton
+import triton.language as tl
+import logging
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic
+@triton.jit
+def where_func(condition, x, y):
+    return tl.where(condition, x, y)
+
+
+def where(condition, x, y):
```

Comment:
where_self?

Reply:
Are we missing where(Tensor condition) in this PR?

Reply:
yep，all cases will be finished in the following commit